bivt-cap/bivt-frontend
# Plugins/expenses/issue26

**### Adds the #expense plugin to dashboard:**
Test Steps:

1. Add a bill
2. Add a bill by choosing different category
3. Change the date to previous week or previous month
4. Check if the tabs are filtering bills correctly
5. Delete a bill
6. Check if the total amounts are correct or not.

Known Issues:

- The order of bills
- The Budget part is under progress 

Changed Files:

- src\components\plugins\expenseManager
- src\redux\plugins\expenseManager
